plugins/@grouparoo/hubspot/__tests__/utils/nockHelper.ts

```diff
@@ -10,31 +10,45 @@ const realPath = path.join(dirPath, ".env");
 function readEnv(filePath) {
   return dotenv.parse(fs.readFileSync(filePath));
 }
-export function loadAppOptions(newNock: boolean = false): SimpleAppOptions {
+export function loadAppOptions(
+  newNock: boolean = false,
+  options?: { useOAuth?: boolean }
+): SimpleAppOptions {
   let envFile;
   if (newNock) {
     envFile = realPath;
   } else {
     envFile = nockPath;
   }
   const parsed = readEnv(envFile);
-  return {
-    hapikey: parsed.HUBSPOT_API_KEY,
-  };
+
+  return options?.useOAuth
+    ? { refreshToken: parsed.HUBSPOT_OAUTH_REFRESH_TOKEN }
+    : { hapikey: parsed.HUBSPOT_API_KEY };
 }
 
 const host = function (url) {
   return new URL(url).host;
 };
+
 export const updater = {
   rewrite: function (nockCall) {
     const realEnv = readEnv(realPath);
     const nockEnv = readEnv(nockPath);
 
-    nockCall = nockCall.replace(
-      new RegExp(realEnv.HUBSPOT_API_KEY, "gi"),
-      nockEnv.HUBSPOT_API_KEY
-    );
+    nockCall = nockCall
+      .replace(
+        new RegExp(realEnv.HUBSPOT_API_KEY, "gi"),
+        nockEnv.HUBSPOT_API_KEY
+      )
+      .replace(
+        new RegExp(realEnv.HUBSPOT_OAUTH_REFRESH_TOKEN, "gi"),
+        nockEnv.HUBSPOT_OAUTH_REFRESH_TOKEN
+      )
+      .replace(
+        new RegExp(process.env.GROUPAROO_AUTH_URL, "gi"),
+        "https://auth.grouparoo.com"
```

Comment:
We could follow the existing pattern and make it clear to people in the future that you'll likely have to set this by putting `GROUPAROO_AUTH_URL` in the hubspot `.env.example`. 

This would then become

```suggestion
        new RegExp(realEnv.GROUPAROO_AUTH_URL, "gi"),
        nockEnv.GROUPAROO_AUTH_URL
```